internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -236,3 +236,24 @@ Resources:
                 Resource: !Sub arn:aws:s3:::${Bucket}/*
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+  
+  EnvManagerS3Access:
+    Metadata:
+      aws:copilot:description: A policy that gives the Env Manager role access to this site's S3 Bucket
+    Type: AWS::IAM::Policy
+    Properties:
+      Roles:
+        - !Sub "${AppName}-${EnvName}-EnvManagerRole"
+      PolicyName: !Sub "${WorkloadName}-S3Access"
```

Comment:
does this need to be unique in the account? if two environments are in the same AWS account then this policy name might clash, if so we need to namespace with !Sub "${AppName}-${EnvName}-${WorkloadName}-S3Access" (and we should limit the number of chars too)

Reply:
discussion here: https://github.com/aws/copilot-cli/pull/4823#discussion_r1183907396